src/screens/Accounts/Accounts.js

```diff
@@ -475,6 +499,9 @@ class AccountsScreen extends React.Component<Props, State> {
 
     const walletsInList = isLegacyUser ? walletsToShow : [smartAccountCard];
 
+    // in case if undefined object comes to array (ref: Sentry issue ID 1292819194)
+    const accountsList = [...walletsInList, ...networksToShow].filter(item => !isEmpty(item));
```

Comment:
How could you get `undefined` here?

Reply:
As per PR description - we're not sure yet. If we put undefined here manually we get exactly same crash as in Sentry, the issue itself is yet to debug.

Reply:
@vyvil finally found an issue, will make another commit re this